data.json

```
@@ -62,5 +62,97 @@
       "factorization",
       "linear-algebra"
     ]
+  },
+  {
+    "command": "",
+    "description": "Plotting API and toolset",
+    "kind": "header",
+    "package": "Plots",
+    "tags": [
+      "GR",
+      "GraphRecipes",
+      "InspectDR",
+      "ORCA",
+      "PGFPlotsX",
+      "PlotlyJS",
+      "PyPlot",
+      "StatsPlots",
+      "UnicodePlots",
+      "plots",
+      "plotting",
+      "visualization"
```

```suggestion
      "plots",
      "plotting",
      "visualization"
```
I think it would be better to leave more general tags.